src/ConnectedField.js

```diff
@@ -208,11 +213,11 @@ const createConnectedField = (structure: Structure<*, *>) => {
     }
 
     handleDragStart = (event: any) => {
-      const { onDragStart, value } = this.props
+      const { name, onDragStart, value } = this.props
```

Comment:
Will this name include full name, including `FormSection`s?
That's probably open to discussion! cc @erikras

Reply:
Yup, it's generated and passed to the field [here](https://github.com/erikras/redux-form/blob/master/src/createField.js#L119).